code/modules/power/cable.dm

```diff
@@ -108,51 +102,72 @@
 /obj/structure/cable/proc/get_powernet()			//TODO: remove this as it is obsolete
 	return powernet
 
+//Telekinesis has no effect on a cable
 /obj/structure/cable/attack_tk(mob/user)
 	return
 
-/obj/structure/cable/attackby(obj/item/W, mob/user, params)
+// Items usable on a cable :
+//   - Wirecutters : cut it duh !
+//   - Cable coil : merge cables
+//   - Multitool : get the power currently passing through the cable
+//
+/obj/structure/cable/attackby(obj/item/W, mob/user)
 
 	var/turf/T = src.loc
 	if(T.intact)
 		return
 
 	if(istype(W, /obj/item/weapon/wirecutters))
-
-//		if(power_switch)
-//			user << "\red This piece of cable is tied to a power switch. Flip the switch to remove it."
-//			return
+///// Z-Level Stuff
+		/* if(src.d1 == 12 || src.d2 == 12)
+			user << "<span class='warning'>You must cut this cable from above.</span>"
+			return */
+///// Z-Level Stuff
+		/* if(breaker_box)
+			user << "\red This cable is connected to nearby breaker box. Use breaker box to interact with it."
+			return */
 
 		if (shock(user, 50))
 			return
 
 		if(src.d1)	// 0-X cables are 1 unit, X-X cables are 2 units long
-			new/obj/item/stack/cable_coil(T, 2, cable_color)
+			new/obj/item/stack/cable_coil(T, 2, color)
 		else
-			new/obj/item/stack/cable_coil(T, 1, cable_color)
+			new/obj/item/stack/cable_coil(T, 1, color)
 
 		for(var/mob/O in viewers(src, null))
-			O.show_message("\red [user] cuts the cable.", 1)
-
+			O.show_message("<span class='warning'>[user] cuts the cable.</span>", 1)
+
+///// Z-Level Stuff
+		/*if(src.d1 == 11 || src.d2 == 11)

```

Comment:
I believe 11 is the magic number for byond's interpration of DOWN. 
I know, this is commented out code, but it should probably use the dir-constant instead of a magic number.